.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish to PyPI
+
+on:
+  push:
+    tags: ["*"]
+
+jobs:
+  pypi-publish:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    - name: create a distribution
+      run: |
+        pip3 install build wheel
+        python3 -m build --wheel
```

Comment:
![Screenshot 2024-11-21 at 17 53 37](https://github.com/user-attachments/assets/bca741d5-2aa4-418c-869c-a722f44d8bff)

We have published torch-dftd with sdist and wheel for the previous versions. Since a wheel distribution covers all the environment that we want to support (only requiring Python3), I think it's enough to publish with one of these formats.

Reply:
I have no strong opinion about this. (Either is fine for me)